shared/config.ts

```diff
@@ -1,8 +1,9 @@
-// eslint-disable-next-line canonical/filename-match-exported
-const contractName = process.env.CONTRACT_NAME || 'near-transactions-accounting-report';
+/* eslint-disable canonical/filename-match-exported */
+const contractName = process.env.CONTRACT_NAME ?? 'near-transactions-accounting-report';
```

Comment:
Why using `??` instead of `||`?

Reply:
https://typescript-eslint.io/rules/prefer-nullish-coalescing/